classes/security/UserGroupDAO.inc.php

```diff
@@ -537,12 +544,16 @@ function getUsersById($userGroupId = null, $contextId = null, $searchType = null
 					' . ($userGroupId ? 'AND ug.user_group_id = ?' : '') . '
 			)';
 		}
-		$sql .= ' ' . $this->_getSearchSql($searchType, $search, $searchMatch, $params);
+		$baseSql .= ' ' . $this->_getSearchSql($searchType, $search, $searchMatch, $params, '');
```

Comment:
Hrm, the difference in behaviour between `null` and `''` is confusing -- but as long as it's cleaner in `main`, I'm OK with it for `stable-3_3_0` and `stable-3_4_0`.

Reply:
It's looking strange for me too! As the `_getSearchSQL()` is private and there's just one caller, I'll update it (remove the order by), which was basically transferred to here...

Reply:
On the `main` branch, this code doesn't exist anymore.